phys/module_surface_driver.F

```diff
@@ -1455,14 +1459,10 @@ SUBROUTINE surface_driver(                                         &
 
    real,    optional,  dimension(ims:ime,jms:jme ),intent(inout) :: XLAIDYN
 ! IRRIGATION
-   INTEGER :: tloc, jmonth,timing  
-   REAL, PARAMETER    :: PI_GRECO=3.14159 
-   INTEGER  :: end_hour, irr_start,xt24,irr_day  
-   REAL :: constants_irrigation   
```

Comment:
OK, this looks like valid clean-up

Reply:
@dudhia @epn09 Why should these lines be removed for this PR?

Reply:
@weiwangncar They are unused variables.

Reply:
@epn09 I understand. But we generally do not want un-related change to be put in a single PR. We may allow it for this time, if @dudhia and @cenlinhe are ok with it. If so, please document the change in your PR.

Reply:
I think as it has no effect, it can be combined but the PR should list it.

Reply:
I agree with Jimy.